src/Services/AccessControl/Interfaces/AccessControlInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\FlowConfig\Services\AccessControl\Interfaces;
+
+use CodeFoundation\FlowConfig\Interfaces\AccessControl\AccessControlInterface as BaseAccessControlInterface;
+
+interface AccessControlInterface extends BaseAccessControlInterface
+{
+    /**
+     * The value is hidden and not returned in bulk gets, but can be retrieved if specifically requested.
+     *
+     * @const int
+     */
+    public const HIDDEN = 1;
```

Comment:
These don't appear to be used yet? I feel like I'm missing something.

Reply:
These are used in Subscriptions, but now thinking about it, they shouldn't be here.